modules/topics-modules.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection.js');
+const format = require('pg-format');
+
+exports.getAllTopics = () => {
+return db.query(format(`SELECT * FROM topics;`))
+    .then((topics) => {
+        if(topics.rows.length !== 0)
+            return topics.rows;
+        else
+            return Promise.reject({status: 500, msg: "Failed to find this DataBase"})
```

Comment:
Topics could be an empty array potentially, that would not mean that the database was not found. At this stage we do not want to reject this promise and should remove the above logig 🙂